data/json/overmap/overmap_special/specials.json

```
@@ -6737,5 +6737,26 @@
     "city_distance": [ 10, 40 ],
     "city_sizes": [ 5, -1 ],
     "flags": [ "CLASSIC", "MAN_MADE", "UNIQUE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "Wind Turbine",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "wind_turbine_ground_north" },
+      { "point": [ 0, 0, 1 ], "overmap": "wind_turbine_pillar_north" },
+      { "point": [ 0, 0, 2 ], "overmap": "wind_turbine_pillar_north" },
+      { "point": [ 0, 0, 3 ], "overmap": "wind_turbine_pillar_platform_north" },
+      { "point": [ 0, 0, 4 ], "overmap": "wind_turbine_pillar_north" },
+      { "point": [ 0, 0, 5 ], "overmap": "wind_turbine_pillar_north" },
+      { "point": [ 0, 0, 6 ], "overmap": "wind_turbine_pillar_north" },
+      { "point": [ 0, 0, 7 ], "overmap": "wind_turbine_pillar_platform_north" },
+      { "point": [ 0, 0, 8 ], "overmap": "wind_turbine_nacelle_north" },
+      { "point": [ 0, 0, 9 ], "overmap": "wind_turbine_roof_north" }
+    ],
+    "connections": [ { "point": [ 0, -1, 0 ], "connection": "local_road", "from": [ 0, 0, 0 ] } ],
+    "locations": [ "field" ],
+    "city_distance": [ 8, 40 ],
+    "occurrences": [ 11, 29 ],
```

Eleven mandatory occurences seems like a mistake, and 29 is also weirdly high for a maximum. The latter is I guess more reasonable as presumably each single overmap special being literally just a single lonesome turbine is something inherited from DDA's implemention, but it will likely be overkill since we have closer-to-functional overmap special placement code.

Might be better to make the min zero and the max lower just to be safe?
```suggestion
    "occurrences": [ 0, 20 ],
```

---

i just used some prime numbers to see if they would work. suggestion commited